src/content/en/tools/puppeteer/debugging.md

```diff
@@ -0,0 +1,11 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Helpful tips for debugging Puppeteer
+
+{# wf_updated_on: 2018-01-10 #}
+{# wf_published_on: 2018-01-10 #}
+{# wf_blink_components: Internals>Headless #}
+
+# Debugging {: .page-title }
+
+{% includecode github_path="GoogleChrome/puppeteer/blob/master/README.md" region_tag="debugging" regexp="## Debugging tips(.*)" %}
```

Comment:
I'm not 100% sure this will render as you expect, we should test on the actual devsite staging server first.

Reply:
Yea definitely. I wasn't sure either. Need to see the real thing.

Reply:
Sent you an email with deets. :(